guides/micronaut-k8s/metadata.json

```diff
@@ -0,0 +1,27 @@
+{
+  "title": "Kubernetes and the Micronaut Framework",
+  "intro": "Use Kubernetes to deploy your Micronaut applications.",
+  "authors": ["Nemanja Mikic"],
+  "tags": ["consul", "validation", "service-discovery"],
+  "categories": ["Micronaut + Kubernetes", "Service Discovery"],
+  "zipIncludes": ["auth.yml", "api/k8s.yml", "orders/k8s.yml", "users/k8s.yml"],
+  "minimumJavaVersion": 17,
+  "publicationDate": "2022-11-23",
+  "apps": [
+    {
+      "name": "users",
+      "features": ["discovery-kubernetes", "management", "security", "kubernetes", "serialization-jackson"],
+      "packageName": "example.micronaut"
```

Comment:
This is the default, isn't it?

Reply:
I added code that if you specify `packageName` inside application block it will use  `name` field as application name. By default all apps are named `micronautguide` and in this example it is required to be named differently because of docker images. I did it in this way to keep it backward compatible with older guides and now other guides if someone wants can name it differently from `example.micronaut`.